src/control/client/connect.go

```diff
@@ -34,6 +34,7 @@ import (
 	mgmtpb "github.com/daos-stack/daos/src/control/common/proto/mgmt"
 	"github.com/daos-stack/daos/src/control/logging"
 	"github.com/daos-stack/daos/src/control/security"
+	"github.com/pkg/errors"
```

Comment:
This import should be in its own group for 3rd-party imports. Seems to have snuck in at the end for a number of files in this PR.

Reply:
that's golangci auto adding imports I'm afraid, I will keep my eye out

Reply:
fixed